packages/services/api/src/modules/preflight-script/module.graphql.ts

```diff
@@ -0,0 +1,53 @@
+import { gql } from 'graphql-modules';
+
+export const typeDefs = gql`
```

Comment:
The module preflight-script is kind of part of the laboratory. So we can merge this module into that folder instead of having a new one.

Reply:
Yes, especially if we ever want to move some pieces of it to GraphiQL as a plugin and embed into Hive Gateway

Reply:
done, should I merge these modules to 1 as well? https://github.com/graphql-hive/console/blob/f02835c164ea71c38b15ffc2b71e67840c876ebf/packages/services/api/src/modules/lab/index.ts#L6-L20

Reply:
that is what I meant.

Reply:
done in https://github.com/graphql-hive/console/pull/5564/commits/c0f6488bad6cd19900aa96b0c23e9e6b293560a9